src/js/store/orgs/reducer.ts

```diff
@@ -164,6 +193,21 @@ const reducer = (
       }
       return orgs;
     }
+    case 'SCRATCH_ORG_COMMIT_CHANGES_FAILED':
+    case 'GITHUB_CHANGES_COMMITTED': {
```

Comment:
Should these events be in the same quasi-namespace?

Reply:
No, for now.

Reply:
> No, for now.

I'm curious why. It seems like `SCRATCH_ORG_COMMIT_CHANGES` would make sense?

Reply:
I guess the frontend does care that it's about an org; I kept seeing it from the backend's perspective, but the event is consumed by the frontend, so.